mocha.el

```diff
@@ -70,7 +70,7 @@ From http://benhollis.net/blog/2015/12/20/nodejs-stack-traces-in-emacs-compilati
 
 (defun mocha-find-project-root ()
   "Find the root of the project."
-  (let ((root-files '(".git" ".hg" ".svn" "package.json")) (dir nil) (i 0))
+  (let ((root-files '("package.json" ".git" ".hg" ".svn")) (dir nil) (i 0))
```

Comment:
is node modules more reliable than package.json? I've never run into a situation where you have one but not the other.